docs/adr/adr_0007.adoc

```diff
@@ -0,0 +1,45 @@
+[[ADR-0007]]
+= ADR-0007: Proposel How to Mark Findings With Hashes to Find Duplicates
+
+[cols="h,d",grid=rows,frame=none,stripes=none,caption="Status",%autowidth]
+|====
+// Use one of the ADR status parameter based on status
+// Please add a cross reference link to the new ADR on 'superseded' ADR.
+// e.g.: {adr_suposed_by} <<ADR-0000>>
+| Status
+| PROPOSED
+
+| Date
+| 2020-11-25
+
+| Author(s)
+| Sven Strittmatter <Sven.Strittmatter@iteratec.com>
+// ...
+|====
+
+== Context
+
+We need the posibility to find duplicate findings. One use case is that we want to accept a finding and wnat to ignore the same findingin the future.
+
+=== Asumptions
+
+* The execution order of hooks is unspecified.
+* The information if a finding's hash is a duplicate is not stored or maintained in the SCB S3 storage.
+* The SCB *does not remove findings*: Read/write hooks may alter them, but never delete or filter them out.
+** Maybe a read hook may decide to not store a finding into an external system.
+
+== Decision
+
+* We generate a hash over each finding so we can compare findings by the hash and identify duplicates.
+* This hash is mutable and may be altered by read/write hooks because we do not wnat to introduce an exception to what a read/write hook may alter.
```

Comment:
```suggestion
* This hash is mutable and may be altered by read/write hooks because we do not want to introduce an exception to what a read/write hook may alter.
```